.circleci/config.yml

```diff
@@ -357,6 +370,12 @@ jobs:
           keys:
             - v6-frontend-admin-dependencies-{{ checksum "~/joanie/src/frontend/admin/yarn.lock" }}
             - v6-frontend-admin-dependencies-
+      - run:
+          name: Setup custom environment variables
+          command: echo 'export NEXT_PUBLIC_API_SOURCE="test"' >> "$BASH_ENV"
```

Comment:
Is it not redundant with the declaration of this variable through the `environment` property ?